### PR TITLE
Add type annotations

### PR DESCRIPTION
**Pull request by lightme16**
- Auto added type annotation. Manually fixed some of them
- Fixed some existing code issues in 2nd commit

### CODE REVIEW

File: tg/colors.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 DEFAULT_FG = curses.COLOR_WHITE
 DEFAULT_BG = curses.COLOR_BLACK
-COLOR_PAIRS = {10: 0}
+COLOR_PAIRS = {(10, 10): 0}
```
Comment (lightme16) [inline tg/colors.py:5]: @paul-nameless was it as bug?

File: tg/main.py [hunk 2]
```diff
@@ -33,8 +33,7 @@ def run(tg: Telegram, stdscr: window) -> None:
     # run this function in thread?
     view = View(stdscr)
     model = Model(tg)
-    controller = Controller(model, view)
-    controller.tg = tg
+    controller = Controller(model, view, tg)
```
Comment (lightme16) [inline tg/main.py:36]: not sure why do we need assign tg dynamically

File: tg/models/__init__.py [hunk 3]
```diff
@@ -19,10 +19,10 @@ def get_me(self):
     def get_user(self, user_id):
         return self.users.get_user(user_id)
 
-    def get_current_chat_msg(self) -> int:
+    def get_current_chat_msg(self) -> Optional[int]:
         chat_id = self.chats.id_by_index(self.current_chat)
         if chat_id is None:
-            return []
```
Comment (lightme16) [inline tg/models/__init__.py:23]: Wrong return value

File: tg/colors.py (hunk 1 above)
Comment (paul-nameless) [reply tg/colors.py:5]: Don't know, I think we can try to remove it and see if color scheme still works fine.

File: tg/main.py (hunk 2 above)
Comment (paul-nameless) [reply tg/main.py:36]: Correct, I fixed it in my patch as well

**Pull request merged**